fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
Pagure is due to be deployed when the mass-rebuild finished committing to dist-git, so either the end of the week or the beginning of next week.

pkgdb will stick around a little longer, but hopefully not more than a week or two, time for things to be ported/deployed.